### ISSUE TITLE
EXP : Fine tuning with different learning rates

### ISSUE THREAD

**Issue by wooyeolbaek** [boostcampaitech4lv23cv1/level2_dataannotation_cv-level2-cv-01#27]
Description
> ICDAR17로 학습한 pretrained weight에 handwriting + upstage rare + 학습에 사용된 ICDAR17의 train중 450장만 추린 것으로 finetuning시 learning rate에 다른 loss 변화

Details
- original learning rate의 변경해가며 각각 비교

Reference / Image
